src/redux/middleware/auth.ts

```diff
@@ -11,7 +12,15 @@ import { history } from '../store';
 export const authMiddleware: Middleware =
   (api: MiddlewareAPI) => (next) => (action) => {
     if (userEndpoints.getUser.matchFulfilled(action)) {
+      if (!Cookies.get('sbUserId')) {
+        Cookies.set('sbUserId', String(action.payload.id));
+      }
+
       api.dispatch(setAuth(true));
+    } else if (userEndpoints.getUser.matchRejected(action)) {
+      if (Cookies.get('sbUserId')) {
```

Comment:
🧐
Условия можно объединить, так как внутри нет `else`-веток
```suggestion
    } else if (userEndpoints.getUser.matchRejected(action) && Cookies.get('sbUserId')) {
```